tests/Backend/ExternalBuckets/BigqueryRegisterBucketTest.php

```diff
@@ -312,11 +324,15 @@ public function testRegisterTableWithLongName(): void
         $tables = $testClient->listTables($idOfBucket);
         $this->assertCount(1, $tables);
 
-        $this->assertCount(1, $refreshJobResult['warnings']);
+        $this->assertCount(2, $refreshJobResult['warnings']);
         $this->assertSame(
             '\'TableTestLongTableTestLongTableTestLongTableTestLongTableTestLongTableTestLongTableTestLongTableTestLong\' is more than 96 characters long',
             $refreshJobResult['warnings'][0]['message'],
         );
+        $this->assertSame(
```

Comment:
akorat to spoleha na to, ze poradi prijde presne takove. to se nam zrovna minulej tyden vymstilo s tema metadatama. Imho bych to treba nejdriv seradil podle abecedy. Ale klidne to tak muzeme nechat